subprojects/android-test/test-inhouse-runner/src/main/kotlin/com/avito/android/runner/InHouseInstrumentationTestRunner.kt

```diff
@@ -291,8 +297,10 @@ abstract class InHouseInstrumentationTestRunner :
 
     fun tryToReportUnexpectedIncident(incident: Throwable) {
         try {
-            report.registerIncident(AppCrashException(incident))
-            report.reportTestCase()
+            if (!report.isWritten) {
+                report.registerIncident(AppCrashException(incident))
+                report.reportTestCase()
+            }
```

Comment:
What else? Will we lost the incident?

Reply:
It is basically any exceptions thrown after test report already written
these exceptions already logged in caller methods